### PR TITLE
Adding Cardanodo 

### PR DESCRIPTION
**Pull request by cardanodo**

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -1,4 +1,19 @@
...
+      "poolId":"20c00ebe39ed12dbd2f3e84a5b57b85c972d1039cf79c70d893b6227",
+      "social":{
+         "facebook":"",
+         "twitter":"NodoPool",
+         "website":"https://cardanodo.com",
+         "reddit":"Cardanodo",
+         "instagram":"",
+         "youtube":""
+      },
+      "telegram":"cardanodo",
```
Comment (leo42) [inline registry.json:13]: We requite a user telegram handle(not groups or channels)_

File: registry.json [hunk 2]
```diff
@@ -1,4 +1,19 @@
...
+      "operatorName":"Story [NODO]",
```
Comment (leo42) [inline registry.json:14]: This needs to be a real name or empty.

**Review by leo42 — CHANGES_REQUESTED** [review]

File: registry.json (hunk 2 above)
Comment (cardanodo) [reply registry.json]: Hi Leo42, 
I have updated my name. My name is Story, and I have included C for my last name.

File: registry.json (hunk 1 above)
Comment (cardanodo) [reply registry.json]: Hi Leo42,
I have updated the telegram handle for my user account.
Thanks,

**Comment by cardanodo** [comment]
Hi Leo42, 
Thanks for taking the time to review our pull request. We have made the changes requested.
Sincerely,
Cardanodo Team

**Review by leo42 — APPROVED** [review]

**Pull request merged**